src/ert/_c_wrappers/enkf/model_config.py

```diff
@@ -17,7 +17,7 @@ class ModelConfig:
 
     def __init__(
         self,
-        num_realizations: int,
+        num_realizations: int = 1,
```

Comment:
since this is cleaning - is it still necessary after introducing a default here having the default `1` in 

```
@classmethod
    def from_dict(cls, refcase: EclSum, config_dict: dict) -> "ModelConfig":
return cls(
                num_realizations=config_dict.get(ConfigKeys.NUM_REALIZATIONS, 1),
                ..
                ..
```

Reply:
The one default is for making it possible to instantiate a minimal config by doing ErtConfig(), without any arguments. The other is for making it possible to have a dict without NUM_REALIZATIONS, which is a change, and should perhaps be removed.